src/components/StudyPane/Passage/WordBlock.tsx

```diff
@@ -98,23 +100,23 @@ export const WordBlock = ({
   
   
     const verseNumStyles = {
-      className: `${zoomLevelMap[ctxZoomLevel].fontSize} top-0 ${ctxIsHebrew ? 'right-0' : 'left-0'} sups w-1 position-absolute ${ctxIsHebrew ? zoomLevelMap[ctxZoomLevel].verseNumMr : zoomLevelMap[ctxZoomLevel].verseNumMl}`
```

Comment:
Since we no longer use zoom level, it doesn't make sense to keep using the zoomLevelMap. We should just keep the font size and other sizes of level 5 and move from a map into an object.

Reply:
If you don't have time, you can just leave it as is. I can refactor the code later

Reply:
I keep the zoom level map here because line 108 to 127 can use smaller font sizes to render words in the uniform width blocks.  Also line 106 uses `zoomLevel + 2` font size for Hebrew. Maybe we can discuss this Saturday then I can refactor this part.